sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizePiiEntitiesTests.cs

```diff
@@ -0,0 +1,169 @@
+
+            PiiEntityCollection entities = await client.RecognizePiiEntitiesAsync(document, "en", new RecognizePiiEntitiesOptions() { DomainFilter = PiiEntityDomainType.ProtectedHealthInformation } );
+
+            Assert.AreEqual(1, entities.Count);
+            Assert.AreEqual("atest@microsoft.com", entities.FirstOrDefault().Text);
+            Assert.AreEqual(EntityCategory.Email, entities.FirstOrDefault().Category);
+            Assert.IsNotNull(entities.RedactedText);
+        }
+
+        [Test]
+        public async Task RecognizePiiEntitiesBatchWithErrorTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            var documents = new List<string>
+            {
+                "A developer with SSN 859-98-0987 whose phone number is 800-102-1100 is building tools with our APIs.",
+                "",
+                "Your ABA number - 111000025 - is the first 9 digits in the lower left hand corner of your personal check.",
+            };
+
+            RecognizePiiEntitiesResultCollection results = await client.RecognizePiiEntitiesBatchAsync(documents);
+
+            Assert.IsTrue(!results[0].HasError);
+            Assert.IsTrue(!results[2].HasError);
+
+            var exceptionMessage = "Cannot access result for document 1, due to error InvalidDocument: Document text is empty.";
+            Assert.IsTrue(results[1].HasError);
+            InvalidOperationException ex = Assert.Throws<InvalidOperationException>(() => results[1].Entities.GetType());
+            Assert.AreEqual(exceptionMessage, ex.Message);
+        }
+
+        [Test]
+        public async Task RecognizePiiEntitiesBatchConvenienceTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            var documents = batchConvenienceDocuments;
+
+            RecognizePiiEntitiesResultCollection results = await client.RecognizePiiEntitiesBatchAsync(documents);
+
+            foreach (RecognizePiiEntitiesResult result in results)
+            {
+                Assert.GreaterOrEqual(result.Entities.Count(), 2);
+            }
+        }
+
+        [Test]
+        public async Task RecognizePiiEntitiesBatchConvenienceWithStatisticsTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            var documents = batchConvenienceDocuments;
+
+            RecognizePiiEntitiesResultCollection results = await client.RecognizePiiEntitiesBatchAsync(documents, "en", new RecognizePiiEntitiesOptions { IncludeStatistics = true });
+
+            foreach (RecognizePiiEntitiesResult result in results)
+            {
+                Assert.GreaterOrEqual(result.Entities.Count(), 2);
+            }
+
+            Assert.IsNotNull(results.Statistics.DocumentCount);
+            Assert.IsNotNull(results.Statistics.InvalidDocumentCount);
+            Assert.IsNotNull(results.Statistics.TransactionCount);
+            Assert.IsNotNull(results.Statistics.ValidDocumentCount);
```

Comment:
I think these are non-nullable integers, right? If that's the case, we can remove these assertions, or making them more specific (like >= 0), if applicable.